shared/components/travel/travel.component.js

```diff
@@ -21,17 +21,18 @@ class TravelComponent extends Panel {
     travel.state = Object.assign({
       simple: true,
       consumption_unit: 'mpg',
-      vehicles: [
-        new Vehicle(travel.newVehicleParams(1), travel, 'mpg'),
-        new Vehicle(travel.newVehicleParams(2), travel, 'mpg')
-      ]
+      vehicles: travel.createVehicles()
     }, travel.userApiState());
   }
 
   get vehicles(){
     return this.state.vehicles;
   }
 
+  get amount_of_vehicles(){
+    return parseInt(this.userApiValue('input_footprint_transportation_num_vehicles'));
+  }
+
   get vehicles_maxed(){
```

Comment:
ok good catch here.

I'd name it `number_of_vehicles` - a little easier to read.